scripts/new-config.sh

```diff
@@ -28,9 +28,7 @@ cat > examples/local/${CONFIG_NAME}.json <<'EOF'
     "shared_filesystem": "",
     "use_gcs_key_secret": false,
     "use_tensorboard": false,
-    "user": "root",
-    "tf_image": "mesosphere/dcos-tensorflow:v1.3",
-    "gpu_tf_image": "mesosphere/dcos-tensorflow:v1.3-gpu"
+    "user": "root"
```

Comment:
For a follow-up we may want to consider looking into whether the default user `nobody` could also be used?

Reply:
@springle Do you know?

Reply:
I don't know much about this field, but I will test the framework with `"user": "nobody"` and make a separate PR if there are no issues with the switch.